### PR TITLE
Typo changes

### PR DESCRIPTION
**Pull request by iobik**
Fixed some typos.

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
     "@types/jwt-decode": "^2.2.1",
-    "axios": "^0.19.2",
+    "axios": "^0.21.1",
```
Comment (soichih) [inline ui/package.json:12]: did you notice any issue with axios 0.19?

Comment (iobik) [reply ui/package.json:12]: No. You can ignore. Thanks

Comment (soichih) [reply ui/package.json:12]: can you amend the PR?

**Comment by soichih** [comment]
Looks like something broke due to the force-push.

```
error: Pulling is not possible because you have unmerged files.
hint: Fix them up in the work tree, and then use 'git add/rm <file>'
hint: as appropriate to mark resolution and make a commit.
fatal: Exiting because of an unresolved conflict.
```

I will fix this later.

**Pull request merged**